.changeset/bright-lobsters-behave.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/infinite-scroll-area": major
+---
```

Comment:
I think you need to add “@yamada-ui/theme”.

Reply:
I fixed it!